Shoko.Server/Providers/AniDB/UDP/AniDBSocketHandler.cs

```diff
@@ -22,6 +22,7 @@ public class AniDBSocketHandler : IAniDBSocketHandler
     private readonly SemaphoreSlim _semaphore = new SemaphoreSlim(1, 1);
     private bool Locked { get; set; }
     public bool IsLocked => Locked;
+    private static int TimeoutMs => 30000; // 30 seconds
```

Comment:
Maybe add one timeout for send and one for receive? They can both be set to 30 seconds for now, but it will make it easier to configure them individually in the future, if the need arise.

Reply:
Will action in next commit.